testutil/ciao-down/cloudinit.go

```diff
@@ -232,3 +232,146 @@ const metaDataTemplate = `
+   permissions: '0755'
+ - content: |
+     deb https://apt.dockerproject.org/repo ubuntu-xenial main
+   path: /etc/apt/sources.list.d/docker.list
+
+runcmd:
+ - echo "127.0.0.1 singlevm" >> /etc/hosts
+ - mount hostgo
+ - chown {{.User}}:{{.User}} /home/{{.User}}
+ - rm /etc/update-motd.d/10-help-text /etc/update-motd.d/51-cloudguest
+ - rm /etc/update-motd.d/90-updates-available
+ - rm /etc/legal
+ - curl -X PUT -d "Booting VM" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "OK" .}}
+{{if len $.HTTPProxy }}
+ - echo "HTTP_PROXY={{.HTTPProxy}}" >> /etc/environment
+ - echo "http_proxy={{.HTTPProxy}}" >> /etc/environment
+{{end -}}
+{{- if len $.HTTPSProxy }}
+ - echo "HTTPS_PROXY={{.HTTPSProxy}}" >> /etc/environment
+ - echo "https_proxy={{.HTTPSProxy}}" >> /etc/environment
+{{end}}
+{{- if or (len .HTTPSProxy) (len .HTTPProxy) }}
+ - echo no_proxy="{{if len .NoProxy}}{{.NoProxy}},{{end}}singlevm"  >> /etc/environment
+{{end}}
+
+ - curl -X PUT -d "Downloading Go" 10.0.2.2:{{.HTTPServerPort}}
+ - echo "GOPATH={{.GoPath}}" >> /etc/environment
+ - echo "PATH=$PATH:/usr/local/go/bin:{{$.GoPath}}/bin"  >> /etc/environment
+ - {{template "PROXIES" .}}wget https://storage.googleapis.com/golang/go1.7.4.linux-amd64.tar.gz -O /tmp/go1.7.4.linux-amd64.tar.gz
+ - {{template "CHECK" .}}
+ - curl -X PUT -d "Unpacking Go" 10.0.2.2:{{.HTTPServerPort}}
+ - tar -C /usr/local -xzf /tmp/go1.7.4.linux-amd64.tar.gz
+ - {{template "CHECK" .}}
+ - rm /tmp/go1.7.4.linux-amd64.tar.gz
+
+ - groupadd docker
+ - sudo gpasswd -a {{.User}} docker
+ - curl -X PUT -d "Installing apt-transport-https and ca-certificates" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get install apt-transport-https ca-certificates
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Add docker GPG key" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Retrieving updated list of packages" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get update
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Upgrading" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get upgrade -y
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Installing Docker" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get install docker-engine -y
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Installing GCC" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get install gcc -y
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Installing Make" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get install make -y
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Installing QEMU" 10.0.2.2:{{.HTTPServerPort}}
+ - {{template "PROXIES" .}}apt-get install qemu-system-x86 -y
+ - {{template "CHECK" .}}
+
+ - curl -X PUT -d "Installing xorriso" 10.0.2.2:{{.HTTPServerPort}}
```

Comment:
@markdryan I kept it around if we needed it for some sort of unit testing. But not needed today.